README.md

```diff
@@ -204,6 +204,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [kemal-heroku-example](https://github.com/cagataycali/kemal-heroku-example) - This repository shows, how you can publish your open source apps which powered kemal framework publish as heroku app in seconds
  * [kemal-mysql-blog](https://github.com/codenoid/kemal-mysql-blog) - Blog written with Crystal, Kemal and MySQL
  * [kemal-react-chat](https://github.com/f/kemal-react-chat) - Build Realtime Web applications with Kemal and React
+ * [kemal-vue-chat](https://github.com/ChangJoo-Park/kemal-vue-chat) - Build Realtime Web applications with Kemal and Vue.js (inspired by awesome above) 
```

Comment:
Nice addition 👍 

Please remove `(inspired by awesome above)` as this may become stale. And please avoid trailing spaces at the end (caught by `crystal spec`)

Reply:
I removed parenthesis :) and trailing space!